src/main/java/core/basesyntax/validator/ValidatorImpl.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.validator;
+
+import core.basesyntax.validator.length.ValidateLineLength;
+import core.basesyntax.validator.length.ValidateLineLengthImpl;
+import core.basesyntax.validator.quantity.QuantityValidator;
+import core.basesyntax.validator.quantity.QuantityValidatorImpl;
+import core.basesyntax.validator.type.TypeValidator;
+import core.basesyntax.validator.type.TypeValidatorImpl;
+
+import java.util.Set;
+
+public class ValidatorImpl implements Validator {
+    ValidateLineLength validateLineLength;
+    TypeValidator typeValidator;
+    QuantityValidator quantityValidator;
+
+    public ValidatorImpl() {
+        validateLineLength = new ValidateLineLengthImpl();
+        typeValidator = new TypeValidatorImpl();
+        quantityValidator = new QuantityValidatorImpl();
+    }
```

Comment:
earlier I wrote that you should pass all fields to the constructor
don't forget about this